### PR TITLE
unclutter: initial package

### PR DESCRIPTION
**Pull request by awiouy**
@islipfd19 This builds and runs on my RPi3. Would you known how to wire it to Chromium?

### CODE REVIEW

**Comment by islipfd19** [comment]
Unclutter needs to be added as a dependency in chromium.

OLD: PKG_DEPENDS_TARGET="toolchain pciutils dbus libXcomposite libXcursor libXtst alsa-lib bzip2 yasm nss libXScrnSaver libexif ninja:host libpng harfbuzz atk gtk+ libva-vdpau-driver"

NEW: PKG_DEPENDS_TARGET="toolchain pciutils dbus libXcomposite libXcursor libXtst alsa-lib bzip2 yasm nss libXScrnSaver libexif ninja:host libpng harfbuzz atk gtk+ libva-vdpau-driver unclutter"

But I'm not sure how to link it to chromium.

**Comment by awiouy** [comment]
Yes, this, and the binary needs to be included in the addon... More is however needed...
@lrusak, could/should unclutter be called from chromium's default.py, and if so, how?

File: packages/addons/browser/chromium/source/resources/settings.xml [hunk 1]
```diff
@@ -10,6 +10,7 @@
         <setting id="HOMEPAGE" type="text" label="1025" default="https://libreelec.tv" />
         <setting id="USE_CUST_AUDIODEVICE" type="bool" label="1026" default="false" />
         <setting id="CUST_AUDIODEVICE_STR" type="text" label="1027" visible="eq(-1,true)" subsetting="true" default="" />
+        <setting id="UNCLUTTER" type="bool" label="1028" default="false" />
```
Comment (chewitt) [inline packages/addons/browser/chromium/source/resources/settings.xml]: If the add-on setting is "Hide Cursor" perhaps `"HIDE_CURSOR_UNCLUTTER"` would be a better name so there's a clear connection between the setting and the `if [ "$UNCLUTTER" == "true" ]` statements?

**Comment by awiouy** [comment]
I have tested this, and, as expected, the cursor does not appear at startup and disappears after a period of inactivity. I will rename the variable to HIDE_CURSOR and rebase the commits tonight.

**Comment by awiouy** [comment]
Please commit

**Pull request merged**